### ISSUE TITLE
Add redirects to ensure permalinks from old site don't break?

### ISSUE THREAD

**Issue by toolness** [18F/pclob#39]
I'm not sure if this is part of the SOW or not, but we might want to consider it. The [USWDS docs repo](https://github.com/18F/web-design-standards-docs) uses the [jekyll-redirect-from](https://github.com/jekyll/jekyll-redirect-from) gem for this.

**Comment by yowill** [comment]
Will Slack says, "the PCLOB site can be launched at whatever URL PCLOB can set DNS for" will this complete this issue once they set the DNS redirect @hbillings  (Did we change the url structure?)

**Comment by toolness** [comment]
@yowill yeah, we changed the URL structure a bit.  I tried to keep as many permalinks as I could, but there are anomalies--for instance, the new site combines the newsroom and events pages into a single "events and press" page, so by definition at least one of the old pages doesn't exist anymore.

Since I think the existing site has analytics set up, we could potentially just look at the top hits on it and ensure that they either still exist in the new URL structure or are redirected somehow.

**Comment by toolness** [comment]
Here's some of the most popular pages according to GA:

|                                                                         |           |                  |                   |           |             |         |            |
|-------------------------------------------------------------------------------------------------------------------|-----------|------------------|-------------------|-----------|-------------|---------|------------|
|                                                                         |           |                  |                   |           |             |         |            |
|                                                                                                                   |           |                  |                   |           |             |         |            |
| Page                                                                                                              | Pageviews | Unique Pageviews | Avg. Time on Page | Entrances | Bounce Rate | % Exit  | Page Value |
| pclob.gov/                                                                                                        | 1,918     | 1,530            | 00:02:41          | 1,502     | 47.05%      | 47.97%  | $0.00      |
| pclob.gov/about-us.html                                                                                           | 563       | 410              | 00:01:22          | 138       | 63.57%      | 41.74%  | $0.00      |
| pclob.gov/library.html                                                                                            | 460       | 264              | 00:02:23          | 60        | 38.89%      | 32.39%  | $0.00      |
| pclob.gov/about-us/board.html                                                                                     | 414       | 312              | 00:00:29          | 35        | 26.47%      | 14.49%  | $0.00      |
| pclob.gov/about-us/staff.html                                                                                     | 366       | 307              | 00:01:54          | 122       | 83.61%      | 49.45%  | $0.00      |
| pclob.gov/contacts.html                                                                                           | 300       | 222              | 00:00:57          | 27        | 71.43%      | 35.67%  | $0.00      |
| pclob.gov/newsroom.html                                                                                           | 285       | 141              | 00:01:02          | 17        | 23.53%      | 13.68%  | $0.00      |
| pclob.gov/events.html                                                                                             | 284       | 163              | 00:00:58          | 15        | 33.33%      | 16.20%  | $0.00      |
| pclob.gov/about-us/board/collins.html                                                                             | 257       | 220              | 00:00:59          | 93        | 67.74%      | 45.91%  | $0.00      |
| pclob.gov/index.html                                                                                              | 236       | 159              | 00:00:55          | 33        | 51.52%      | 30.08%  | $0.00      |
| pclob.gov/opportunities.html                                                                                      | 217       | 167              | 00:00:30          | 22        | 54.55%      | 23.50%  | $0.00      |
| pclob.gov/about-us/board/brand.html                                                                               | 171       | 141              | 00:01:38          | 81        | 81.18%      | 56.73%  | $0.00      |
| pclob.gov/about-us/board/medine.html                                                                              | 106       | 91               | 00:00:43          | 32        | 71.88%      | 36.79%  | $0.00      |
| pclob.gov/about-us/board/wald.html                                                                                | 96        | 77               | 00:00:44          | 34        | 82.35%      | 40.62%  | $0.00      |
| pclob.gov/about-us/board/dempsey.html                                                                             | 92        | 75               | 00:01:19          | 26        | 73.08%      | 35.87%  | $0.00      |
| pclob.gov/newsroom/20160205.html                                                                                  | 64        | 43               | 00:03:23          | 2         | 25.00%      | 31.25%  | $0.00      |
| pclob.gov/foia/foia.html                                                                                          | 35        | 27               | 00:01:05          | 8         | 11.11%      | 31.43%  | $0.00      |
| pclob.gov/events/2014/july02.html                                                                                 | 32        | 24               | 00:03:19          | 4         | 0.00%       | 28.12%  | $0.00      |
| pclob.gov/website-privacy-policy.html                                                                             | 30        | 15               | 00:01:00          | 4         | 40.00%      | 13.33%  | $0.00      |
| pclob.gov/events/2014/january23.html                                                                              | 24        | 20               | 00:01:40          | 3         | 33.33%      | 54.17%  | $0.00      |
| pclob.gov/newsroom/20160329.html                                                                                  | 19        | 19               | 00:01:34          | 11        | 90.91%      | 63.16%  | $0.00      |
| pclob.gov/newsroom/20160413.html                                                                                  | 18        | 13               | 00:00:28          | 5         | 25.00%      | 33.33%  | $0.00      |
| pclob.gov/newsroom/20160211.html                                                                                  | 12        | 12               | 00:00:00          | 10        | 100.00%     | 100.00% | $0.00      |
